way_to_home/custom_user/views.py

```diff
@@ -116,3 +125,53 @@ def signin_google(request):
         return HttpResponse("User was successfully created", status=201)
 
     return HttpResponse("User's data is empty", status=400)
+
+
+@require_http_methods(["POST"])
+def reset_password(request):
+    """Function that provides reset user password"""
+    data = request.body
+    if not update_email_validator(data, 'email'):
+        return False
+    email = data.get('email')
+    user = CustomUser.get_by_email(email=email)
+    if not user:
+        return RESPONSE_400_INVALID_DATA
+    token = create_token(data={'email': user.email})
+    send_email_password_update(user, token)
+    return RESPONSE_200_OK
+
+
+@require_http_methods(['PUT'])
+def confirm_reset_password(request, token):
+    """Function that provides confirm reset user password"""
+    confirm = decode_token(token)
+    if not confirm:
+        return RESPONSE_498_INVALID_TOKEN
+    user = CustomUser.get_by_email(email=confirm.get('email'))
+    if not user:
+        return RESPONSE_400_OBJECT_NOT_FOUND
+
+    data = request.body
+    if not reset_password_validator(data, 'new_password'):
+        return False
```

Comment:
Same situation as mentioned above.
You can just use `password_validator`. I think it is better when you have only one required field.
`password_validator` also handles the case when the value is `None`.

Reply:
done